packages/twitch/src/API/Kraken/Team/TeamWithUsers.ts

```diff
@@ -0,0 +1,23 @@
+import Team, {TeamData} from './Team';
+import User, {UserData} from '../User/User';
+import TwitchClient from '../../../TwitchClient';
+
+/** @private */
+export interface TeamUsersData extends TeamData {
+	users: UserData[];
+}
+
+export default class TeamWithUsers extends Team {
+	/** @private */
+	constructor(/** @private */ protected _data: TeamUsersData, client: TwitchClient) {
+		super(_data, client);
+	}
+
+	/**
+	 * The list of users in the team.
+	 */
+	async users(): Promise<User[]> {
```

Comment:
As mentioned above. Although you can keep the originally synchronous getter and just have the async method as an alias to it.